SevenSlots/SevenSlots/SevenSlots/Commands/LoginCommand.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Rg.Plugins.Popup.Services;
+using SevenSlots.Model;
+using SevenSlots.Services;
+using SevenSlots.View;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace SevenSlots.Commands
+{
+    internal class LoginCommand : ICommand
+    {
+        public event EventHandler CanExecuteChanged;
+        IDatabaseService databaseService;
+
+        public LoginCommand()
+        {
+            databaseService = DependencyService.Get<IDatabaseService>();
+        }
+
+        public bool CanExecute(object parameter)
+        {
+            return true;
+        }
+
+        public async void Execute(object parameter)
+        {
+            List<User> users = await databaseService.getAllUsers();
+            var result = users.FirstOrDefault(u => u.Username == (parameter as User).Username &&
+                             u.Password == (parameter as User).Password);
```

Comment:
There is no point to process the database results here. Please make this a method in the databaseService.

Reply:
*Quick note:
I would like to have this as an endpoint in the backend so we don't have access to the list of users at all (please).

Reply:
i created a task for this